controller/users.go

```diff
@@ -600,6 +625,104 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 	return returnResponse
 }
 
+func (c *UsersController) createAuthUpdateClient(ctx *app.UpdateUsersContext) (*authservice.Client, error) {
```

Comment:
fine for now, but we should consolidate the client initialization methods somewhere in future, for re-use.